src/uct/ib/rc/base/rc_ep.h

```diff
@@ -44,6 +44,9 @@ enum {
      * are needed */
     UCT_RC_EP_FLAG_KEEPALIVE_PENDING = UCS_BIT(0),
 
+    /* EP connect to EP mode is used, ready to send */
+    UCT_RC_EP_FLAG_CONNECT_TO_EP     = UCS_BIT(1),
```

Comment:
maybe just UCT_RC_EP_FLAG_CONNECTED?
RC would always be "connect to ep" mode

Reply:
renamed to UCT_RC_EP_FLAG_CONNECTED